oracle-server-ui/package.json

```diff
@@ -1,12 +1,13 @@
 {
   "name": "oracle-server-ui",
-  "version": "0.0.2",
+  "version": "0.0.3",
   "scripts": {
     "ng": "ng",
     "start": "ng serve",
-    "startproxy": "ng serve --proxy-config proxy.conf.json",
+    "startproxy": "ng serve --watch --proxy-config proxy.conf.json",
     "build": "ng build",
     "watch": "ng build --watch --configuration development",
+    "watchproxy": "ng build --watch --configuration development --proxy-config proxy.conf.json",
     "test": "ng test"
```

Comment:
Do these tests pass for you? This is what i get when i try to run

<img width="1196" alt="Screen Shot 2021-09-18 at 12 39 08 PM" src="https://user-images.githubusercontent.com/3514957/133897634-6e261602-8416-47df-aa8b-bb76b9904f75.png">

<img width="1196" alt="Screen Shot 2021-09-18 at 12 40 09 PM" src="https://user-images.githubusercontent.com/3514957/133897645-2b5f86a5-67b3-47dd-8ffd-0efbc8add585.png">

Reply:
I have the same experience on my local. There are no meaningful tests currently, so I have only changed what the CI complains about.